src/mediaaccessibility.cs

```diff
@@ -45,4 +46,29 @@ interface MAVideoAccommodations {
 		[Field ("kMADimFlashingLightsChangedNotification")]
 		NSString DimFlashingLightsChangedNotification { get; }
 	}
+
+	[TV (17,0), Mac (14,0), iOS (17,0), MacCatalyst (17,0)]
+	[BaseType (typeof(NSObject))]
+	interface MAFlashingLightsProcessorResult
+	{
+		[Export ("surfaceProcessed")]
+		bool SurfaceProcessed { get; }
+
+		[Export ("mitigationLevel")]
+		float MitigationLevel { get; }
+
+		[Export ("intensityLevel")]
+		float IntensityLevel { get; }
+	}
+
+	[TV (17,0), Mac (14,0), iOS (17,0), MacCatalyst (17,0)]
+	[BaseType (typeof(NSObject))]
+	interface MAFlashingLightsProcessor
+	{
+		[Export ("canProcessSurface:")]
+		bool CanProcess (Surface surface);
+
+		[Export ("processSurface:outSurface:timestamp:options:")]
+		MAFlashingLightsProcessorResult Process (Surface inSurface, Surface outSurface, double timestamp, [NullAllowed] NSDictionary<NSString, NSObject> options);
```

Comment:
Can the options be a strong dictionary?

Reply:
I did not find any documentation about the keys or values. https://developer.apple.com/documentation/mediaaccessibility/maflashinglightsprocessoroptionkey?language=objc

Or in the header: https://github.com/xamarin/xamarin-macios/wiki/MediaAccessibility-iOS-xcode15.0-b1

Reply:
FYI - I also looked at the Swift doc'n and the key, oddly enough, is *not* an `NSString`, which is an inner struct called `MAFlashingLightsProcessor.OptionKey` which has initializers from `String` - it feels like they're keeping their options open for the future, but the values of the options are also untyped there and undocumented.

Reply:
I did notice the same, we should leave it as open as possible.